declarations/Canva and Affinity.json

```
@@ -0,0 +1,57 @@
+{
+  "name": "Canva and Affinity",
+  "terms": {
+    "Privacy Policy": {
+      "fetch": "https://www.canva.com/policies/privacy-policy/",
+      "select": "div[class=HqfB2mG]"
```

For selectors as well, you can just use the `.className` notation. It keeps it simple. Now I have had a look at the terms and this seems to be a good selector pairing that we can use in this case. We try as much as possible to look for alternative selectors that are simple and not system generated. You can target a wider region then remove unnecessary sections, if there is no single simple selector that can target the desired section

You can have a look at this suggestion and if it looks good you can replicate it throughout all the other documents as they have a similar structure 

```suggestion
      "select": "#__next",
      "remove": [
        "header",
        "footer",
        ".sW0ID7p"
      ]
```